androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/EnhancedVideoPlayer.kt

```diff
@@ -2,31 +2,48 @@ package com.profusion.androidenhancedvideoplayer.components
 
 import android.content.res.Configuration.ORIENTATION_LANDSCAPE
 import android.net.Uri
-import androidx.compose.foundation.clickable
-import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.animation.* // ktlint-disable no-wildcard-imports
```

Comment:
nit: lint

Reply:
I think we will have to disable this rule to reduce the number of imports, this file tends to be very large and therefore the number of imports too. in anova project we disabled this rule.

Reply:
I agree with you @thalysmcarrara, but if you want to disable this rule, I'd prefer to disable it for the entire project

Reply:
i can create a a new issue for that, because its not part of the current issue

Reply:
It's not part of the current issue to add `// ktlint-disable` as well. We are not disabling rules in any part of the codebase. So, you either add a one-line configuration to the `.editorconfig` and remove the `// ktlint-disable` or adopt the ktlint rules of the project.

Reply:
I added the lint rule in a .editorconfig file, I thought it would be more work